tap_asana/streams/portfolios.py

```diff
@@ -40,17 +25,33 @@ class Portfolios(Stream):
     "portfolio_items"
   ]
 
+  @asana_error_handling
+  def get_items_for_portfolio(self, portfolio_gid):
```

Comment:
Why do we move these methods inside the class?

Reply:
We are using `self.request_timeout` as the timeout value in the function. Previously there was no `self` used so we have kept the outside the class. Now the `self.request_timeout` will be used in the function, hence, we moved the function in the class.